packages/storage/src/userStorage.ts

```diff
@@ -235,6 +237,12 @@ export class UserStorage {
     return summary;
   }
 
+  private async getOrCreateBucket(client: Buckets, name: string): Promise<GetOrCreateResponse> {
+    return client.getOrCreate(name, {
+      threadID: getDeterministicThreadID(this.user.identity as PrivateKey).toString(),
```

Comment:
We should create a new thread/db id for each bucket, then save that new id inside a collection inside the deterministic thread.  Reference: https://github.com/FleekHQ/space-daemon/blob/master/core/textile/bucket_factory.go#L116.